src/QMCHamiltonians/BareKineticEnergy.cpp

```diff
@@ -402,6 +403,106 @@ void BareKineticEnergy::evaluateOneBodyOpMatrixForceDeriv(ParticleSet& P,
     }
 }
 
+void BareKineticEnergy::createResource(ResourceCollection& collection) const
+{
+  auto new_res        = std::make_unique<MultiWalkerResource>();
```

Comment:
Everywhere else in the code the "MultiWalkerResource" has a class name attached (e.g. CoulombPBCAAMultiWalkerResource.  Should this also have it?  Alternately, could the name be additionally qualified? (BareKineticEnergy::MultiWalkerResource).   
The issue is just reading through the code it looks like this is a generic class, but it is specific to BareKineticEnergy.

Reply:
You're definitely right, I did a bunch of these and became less thoughtful. I'll stick with the current convention we've used so far. Fully qualifying is tempting but I'm not a big fan of Classes whose meaning changes based on name scopes inside of a project. Maybe we should consider an abbreviation as these types are obviously going to continue to appear.